tests/recipes/utils.py

```diff
@@ -128,6 +128,22 @@ def llama3_test_config() -> List[str]:
     ]
 
 
+def llama3_test_config_v2() -> List[str]:
```

Comment:
Can we have slightly more descriptive name? Like llama3_test_large or llama3_test_128?

Reply:
yeah i got lazy. will update